yb-voyager/cmd/exportData.go

```diff
@@ -481,16 +481,19 @@ func getPGDumpSequencesAndValues() (map[*sqlname.SourceName]int64, error) {
 		}
 		argsIdx := setvalRegex.SubexpIndex("args")
 		if argsIdx > len(matches) {
-			return nil, fmt.Errorf("invalid index %d for matches - %s for line %s", argsIdx, matches, line)
+			return utils.StructMap[sqlname.NameTuple, int64]{}, fmt.Errorf("invalid index %d for matches - %s for line %s", argsIdx, matches, line)
```

Comment:
Once you change the return type to a pointer, return `nil` `StructMap` when there is an error (just like the earlier code did).